tasks/bing_ads_ruby_sdk.rake

```diff
@@ -2,17 +2,19 @@ require 'dotenv/load'
 
 namespace :bing_token do
   desc "Gets and stores Bing OAuth token in file"
-  task :get, [:filename, :bing_developer_token, :bing_client_id] do |task, args|
+  task :get, [:filename, :bing_developer_token, :bing_client_id, :bing_client_secret] do |task, args|
 
     filename = args[:filename] || ENV.fetch('BING_STORE_FILENAME')
     developer_token = args[:bing_developer_token] || ENV.fetch('BING_DEVELOPER_TOKEN')
     bing_client_id = args[:bing_client_id] || ENV.fetch('BING_CLIENT_ID')
+    bing_client_secret = args[:bing_client_secret] || ENV.fetch('BING_CLIENT_SECRET')
```

Comment:
Won't this require that the `bing_client_secret` be provided as an argument or an ENV var to the Rake task?

If you use one of these the Rake task arg will be optional:
* `ENV.fetch('BING_CLIENT_SECRET', nil)`
* `ENV['BING_CLIENT_SECRET']`